### DISCUSSION TITLE
How to diasable the "welcome" bar without logging in?

### DISCUSSION THREAD

**Discussion by HubKing** [Kong/insomnia#6655] [Help] [answered]
This whole part is a waste of space.

![image](https://github.com/Kong/insomnia/assets/10900489/f2919aa7-d474-4b88-a202-c7dbed7d09ac)

**Comment by filfreire** [comment, marked as answer]
@HubKing this welcome bar can be dismissed from version [8.2.0](https://insomnia.rest/changelog#8.2.0) onwards.

https://github.com/Kong/insomnia/releases/tag/core%408.2.0
